tests/python/unittest/test_operator.py

```diff
@@ -3521,14 +3521,6 @@ def test_empty_reps():
         b_tiled = mx.nd.tile(b, ()).asnumpy()
         assert same(a_tiled, b_tiled)
 
-    def test_zero_reps():
-        a = np.array([[2, 3, 4], [5, 6, 7]], dtype=np.int32)
-        b = mx.nd.array(a, ctx=default_context(), dtype=a.dtype)
-        reps = (2, 0, 4, 5)
```

Comment:
why was this supported previously? What was the behavior?

Reply:
It would return an empty NDArray with some dim size equal to zero. I was following the NumPy's convention to implement the behavior. Since MXNet does not support empty NDArrays in any way, I think it's okay to move this to `assert_exception` test?